OpenRA.Mods.Common/Graphics/ContrailRenderable.cs

```diff
@@ -63,8 +65,8 @@ public IRenderable OffsetBy(in WVec vec)
 		public IFinalizedRenderable PrepareRender(WorldRenderer wr) { return this; }
 		public void Render(WorldRenderer wr)
 		{
-			// Need at least 4 points to smooth the contrail over
-			if (length - skip < 4)
+			var renderLength = length - skip;
+			if (renderLength <= 0)
```

Comment:
I assume with a `renderLength` of 1 we will never be able to draw a line, so this should be:
```suggestion
			if (renderLength <= 1)
```
?

Reply:
fixed, but we maybe we need to comment it here about why it must >=1 (and I don't know how to explain on this)